llvm/lib/SYCLLowerIR/LowerInvokeSimd.cpp

```diff
@@ -73,10 +72,6 @@ ModulePass *llvm::createSYCLLowerInvokeSimdPass() {
 
 namespace {
 // TODO support lambda and functor overloads
```

Comment:
Do we need to move this `TODO` as well or is it unrelated to the removed code below?